selftests/unit/test_multiplexer.py

```diff
@@ -39,7 +39,7 @@ def test_basic(self):
         self.assertNotEqual(self.params1, self.params2)
         repr(self.params1)
         str(self.params1)
-        str(multiplexer.AvocadoParams([], 'Unittest', [], {}))
+        str(variants.AvocadoParams([], 'Unittest', [], {}))
```

Comment:
Sure, this commit is marked as FIXME, it's only meant as RFC whether the name Variants would be OK and demonstration how would this change look like in the code.